agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptionsParser.java

```diff
@@ -156,12 +163,39 @@ private void handleOption(AgentOptions options,
 						value)) {
 			return;
 		}
+   		if (key.startsWith("proxy-") && handleProxyOptions(options, StringUtils.stripPrefix(key, "proxy-"), value, filePatternResolver)){
+				return;
+			}
 		if (handleAgentOptions(options, key, value)) {
 			return;
 		}
 		throw new AgentOptionParseException("Unknown option: " + key);
 	}
 
+	private boolean handleProxyOptions(AgentOptions options, String key, String value, FilePatternResolver filePatternResolver) throws AgentOptionParseException {
+		String httpsPrefix = String.format("%s-", ProxySystemProperties.Protocol.HTTPS);
+		if (key.startsWith(httpsPrefix)
+				&& options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTPS).handleTeamscaleProxyOptions(StringUtils.stripPrefix(
+				key, httpsPrefix), value)) {
+			return true;
+		}
+
+		String httpPrefix = String.format("%s-", ProxySystemProperties.Protocol.HTTP);
+		if (key.startsWith(httpPrefix)
+				&& options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTP).handleTeamscaleProxyOptions(StringUtils.stripPrefix(
+				key, httpPrefix), value)) {
+			return true;
+		}
```

Comment:
The way it is written now does not match what we tell the user to use as a parameter in the Readme. 

I just tested it locally and set the config parameter as: 
`proxy-host=localhost:80` but this fails, as the expected parameter would be
`http-proxy-host`. 

Please either update the agent/README.md or change the parameter, so that setting a `proxy-host` just sets it for both http and https.

Reply:
The more I think about it the better I like the fact that we just keep the option as `proxy-host` and set it for both http and https.

Reply:
Tbh I'm not sure I agree. This would "mix" the two different behaviors of the global properties and the teamscale-specific ones, because the current implementation allows to use the global ones as defaults *for a specific protocol*, which are *selectively* overwritten by the Teamscale-specific options.

TLDR I'm not sure if increasing the complexity of the options with separate http-/https-options or the increasing the complexity by differing from the default behavior of the global options is more confusing to the users. 

WDYT?

Reply:
Okay you made a compelling case. Then we should just update the docs so the user knows what to type. ;)